.github/buildomat/jobs/tuf-artifact.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#:
+#: name = "helios / tuf-artifact"
+#: variety = "basic"
+#: target = "helios-latest"
+#: rust_toolchain = "nightly-2022-09-27"
+#: output_rules = [
+#:	"=/work/repo.zip",
+#: ]
+#:
+#: [dependencies.package]
+#: job = "helios / package"
+#:
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+cargo --version
+rustc --version
+
+COMMIT=$(git rev-parse HEAD)
+VERSION="0.0.0+git${COMMIT:0:11}"
+
+pushd /input/package/work/out
+tar cvf "/work/control-plane.tar" ./*
+popd
+
+cargo build --locked --release --bin tufaceous
+
+python3 -c 'import secrets; open("/work/key.txt", "w").write("ed25519:%s\n" % secrets.token_hex(32))'
```

Comment:
What is this key?

Reply:
It's a throwaway key. Perhaps I should make that clearer.

For the purposes of this repository, I don't think we have a good way of establishing trust other than "the repo was spit out of buildomat so it's probably related to this commit".